src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -110,6 +121,7 @@ class DraftJSEditor extends React.Component<Props> {
                         <div>
                             <Editor
                                 {...a11yProps}
+                                ariaLabel={formatMessage(messages.commentTextInputLabel)}
```

Comment:
For context the flow is as following:  CommentForm > DraftJSMentionSelector > DraftJSMentionSelectorCore > DraftJSEditor > actual Editor from draft-js package

adding ariaLabel prop here renders it in the DOM on the comment form